library/java/net/openid/appauth/AuthorizationException.java

```diff
@@ -508,13 +509,15 @@ public static AuthorizationException fromOAuthTemplate(
             @NonNull AuthorizationException ex,
             @Nullable String errorOverride,
             @Nullable String errorDescriptionOverride,
-            @Nullable Uri errorUriOverride) {
+            @Nullable Uri errorUriOverride,
+            @NonNull JSONObject responseJson) {
```

Comment:
This parameter should be `@Nullable` since the static methods above pass `null` to it.

Reply:
Fixed, thanks!